warehouse/query-core/src/main/java/datawave/query/iterator/logic/AndIterator.java

```diff
@@ -124,13 +135,60 @@ public void remove() {
     }
     
     public boolean hasNext() {
-        if (null == includeHeads) {
+        if (!isInitialized()) {
             throw new IllegalStateException("initialize() was never called");
         }
         
         return next != null;
     }
     
+    @Override
+    public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean inclusive) throws IOException {
+        // seek all of the iterators. Drop those that fail, as long as we have at least one include left
+        Iterator<NestedIterator<T>> include = includes.iterator();
+        while (include.hasNext()) {
+            NestedIterator<T> child = include.next();
+            try {
+                for (NestedIterator<T> itr : child.leaves()) {
+                    if (itr instanceof SeekableIterator) {
+                        ((SeekableIterator) itr).seek(range, columnFamilies, inclusive);
+                    }
+                }
+            } catch (Exception e) {
+                include.remove();
+                if (includes.isEmpty()) {
+                    throw e;
+                } else {
+                    log.warn("Failed include lookup, but dropping in lieu of other terms", e);
+                }
+                
+            }
+        }
+        Iterator<NestedIterator<T>> exclude = excludes.iterator();
+        while (exclude.hasNext()) {
+            NestedIterator<T> child = exclude.next();
+            try {
+                for (NestedIterator<T> itr : child.leaves()) {
+                    if (itr instanceof SeekableIterator) {
+                        ((SeekableIterator) itr).seek(range, columnFamilies, inclusive);
+                    }
+                }
+            } catch (Exception e) {
+                exclude.remove();
```

Comment:
I don't think its safe to remove from the excludes list. In NegationFilter.isFiltered() the check is against any filter

Reply:
You are absolutely correct.  Will fix